exercises/armstrong-numbers/canonical-data.json

```diff
@@ -0,0 +1,189 @@
+{
+  "exercise": "armstrong-number",
+  "version": "1.0.0",
+  "comments": [
+    "Both input-parameters are always given in base-10 (decimal)."
+  ],
+  "cases": [
+    {
+      "description": "0 is an Armstrong number in base 10",
+      "property": "armstrong",
+      "input": {
+        "number": 0,
+        "base": 10
+      },
+      "expected": true
+    },
+    {
+      "description": "1 is an Armstrong number in base 10",
+      "property": "armstrong",
+      "input": {
+        "number": 1,
+        "base": 10
+      },
+      "expected": true
+    },
+    {
+      "description": "2 is an Armstrong number in base 10",
+      "property": "armstrong",
+      "input": {
+        "number": 2,
+        "base": 10
+      },
+      "expected": true
+    },
+    {
+      "description": "9 is an Armstrong number in base 10",
+      "property": "armstrong",
+      "input": {
+        "number": 9,
+        "base": 10
+      },
+      "expected": true
+    },
+    {
+      "description": "10 is not an Armstrong number in base 10",
+      "property": "armstrong",
+      "input": {
+        "number": 10,
+        "base": 10
+      },
+      "expected": false
+    },
+    {
+      "description": "100 is not an Armstrong number in base 10",
+      "property": "armstrong",
+      "input": {
+        "number": 100,
+        "base": 10
+      },
+      "expected": false
+    },
+    {
+      "description": "153 is an Armstrong number in base 10",
+      "property": "armstrong",
+      "input": {
+        "number": 153,
+        "base": 10
+      },
+      "expected": true
+    },
+    {
+      "description": "370 is an Armstrong number in base 10",
```

Comment:
I had no specific objection to 111, but rather a general objection to ALL the tests.

The description here doesn't tell me anything already covered by the input and output values.
What is this test testing?
Why is  this number useful to test?
If we had a solution that passed the previous n tests, what code would need to change that would make this test pass?

What is it about each of these numbers that makes them interesting/important to test?

Reply:
I guess its easier if we turn this around:
What numbers do you suggest we test/are interesting to test?